cfp.html

```diff
@@ -16,7 +16,11 @@ <h1 class="c-red">
             <p>You’ll be joining over 16 years of tradition in inspiring and amazing Rubyists and other developers around Europe.</p>
             <p>A few things to note:</p>
             <ul>
+                <li>EuRuKo is a non-profit event.</li>
                 <li>All talks will have to adhere to our <a href="/coc">Code of Conduct</a>.</li>
+                <li>We are fully committed to have a lineup as diverse as we can.</li>
+                <li>As a non-profit event, we cannot offer to pay you for speaking. We can, however, cover travel and accommodation (see the perks section).</li>
+                <li>We will dedicate a few slots for special speakers such as Matz.</li>
```

Comment:
"We will dedicate a few slots for invited speakers, like Matz."

And maybe add to that "Invited speakers are not paid a fee either, but do not have to go through the CfP process." (if true :grin:)

Reply:
Which it is, yes! Thanks!